### PR TITLE
Display the list of autocomplete suggestions

### LINKED ISSUE #197: Display the list of autocomplete suggestions in the start edit view
## Description
We need to react to the list of autocomplete suggestions and display it according to the designs in Invision
Tapping any of the items in that list should dispatch an AutocompleteSuggestionTapped action

## Definition of done
- [ ] The list is displayed in the UI
- [ ] Tapping items in the list dispatch actions accordingly
- [ ] The documentation is updated to reflect the changes done in this PR

### PR DESCRIPTION
**Pull request by Odrakir**
### Summary
This adds the view controller that shows the autocomplete suggestions and adapts its height to the position (top or bottom) and the count of suggestions.

This PR doesn't add the actual designs because those are going to change (we are probably moving to a floating view). Also I think this PR is big enough.

It does handle tapping the suggestion and sending the appropriate action.

### Technical considerations
I'm not using RxDataSources to populate the table view, it might make animations better, so it could be an improvement we can consider.

### Relationships
Closes #197 

### Pull Request checklist
- [ ] The code does not hinder the app's performance
- [ ] The code does not introduce regressions
- [ ] The code in this PR is tested thoroughly enough that changing the behavior to an incorrect one breaks tests

### Pointers
Check one commit at a time

### Review template

- [ ] The code is thoroughly tested in well written and resilient manner
- [ ] The code is performant and leads to an optimal user experience
- [ ] The code is testable and does not cut corners for the sake of delivering
- [ ] The code considers older platforms well and will work in older devices just like it will in new ones

- [ ] The code considers light and dark theme
- [ ] The code considers RTL and does not mention left/right
- [ ] The code considers i18n
  - [ ] There are no hardcoded strings in the UI
  - [ ] All strings are included in every language we support OR there's an issue requesting new translations
- [ ] There are no hard-coded numbers referring to dimensions or font sizes
- [ ] The code considers UI elements that are similar and extracts their similarities into styles to avoid duplication

## Merge permissions
Go!

### CODE REVIEW

File: Modules/Timer/Timer/Common/Views/BottomSheetViewController.swift [hunk 1]
```diff
@@ -131,7 +128,7 @@ class BottomSheetViewController: UIViewController, UIGestureRecognizerDelegate {
         case .partial:
             view.isUserInteractionEnabled = true
             let viewHeight = view.frame.height - view.safeAreaInsets.top - view.safeAreaInsets.bottom
-            topConstraint.constant = viewHeight - minViewHeight - keyboardHeight + dragAmount
+            topConstraint.constant = viewHeight - minViewHeight /*- keyboardHeight*/ + dragAmount
```
Comment (dshr) [inline Modules/Timer/Timer/Common/Views/BottomSheetViewController.swift:131]: :fire:

File: Modules/Timer/Timer/Suggestions/SuggestionsViewController.swift [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+    @IBOutlet weak var tableView: UITableView!
+    private var heightConstraint: NSLayoutConstraint!
+
+    private var suggestions = [AutocompleteSuggestion]()
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+
+        tableView.backgroundColor = UIColor.init(white: 0.9, alpha: 1.0)
+        tableView.rowHeight = rowHeight
```
[the remark this answers is not in the document]
Comment (Odrakir) [reply Modules/Timer/Timer/Suggestions/SuggestionsViewController.swift:25]: I need to know the height of the table view's content to see how heigh the suggestions view needs to be. Making it constraint based might be possible, but much more complicated and probably less performant.

[the remark this answers is not in the document]
Comment (dshr) [reply Modules/Timer/Timer/Suggestions/SuggestionsViewController.swift:25]: Fair enough, let's add a comment there that it has to be changed later on to support dynamic type. I guess we could calculate a ration between font size and rowHeight, and calculate that as part of the styling stuff somehow...

File: Modules/Timer/Timer/Common/Views/BottomSheetViewController.swift (hunk 1 above)
Comment (Odrakir) [reply Modules/Timer/Timer/Common/Views/BottomSheetViewController.swift:131]: Oh! Sorry.

**Pull request merged**